### PR TITLE
Bug 1218666 - Do not change audio channel in SoundManager if content is playing r=alwu

### PR DESCRIPTION
**Pull request by evanxd**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6415704d27302992b0909e044c46639510817f0e

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=781d4383ef7511a2bcfcb90a2397fe69a06c9145

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d0b27d8a73f57d48e4a0f52aa35879b0e0a24cb7

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=fa7aa16b3cce294d5c3d8a964184d8eb1cf06ccb

File: tv_apps/smart-system/js/audio_channel_service.js [hunk 1]
```diff
@@ -43,6 +45,11 @@
      */
     _start: function() {
       this._activeAudioChannels = new Map();
+      this._audioChannelWeights = new Map([
+        ['normal', 1], ['content', 2], ['alarm', 3],
+        ['system', 4], ['ringer', 5], ['telephony', 6],
+        ['notification', 7], ['publicNotification', 8]
+      ]);
```
Comment (alastor0325) [inline tv_apps/smart-system/js/audio_channel_service.js]: I suggest to adjust 'system' type to the lowest weight.

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=51030ac88c47ea3bb1a9efff93212416f0d26ced

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=071a5de4afc3694270582cafdf265f2c526e1f0a

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=f9109be19ed1fd9837f6dbb8752cf7ac242a950d

**Pull request merged**